Extensions/News/Main.js

```diff
@@ -0,0 +1,62 @@
+const Net = require('./NetHandler');
+const fs = require('fs');
+var path = require('path');
+const { F_OK } = require('constants');
```

Comment:
pleeeease let's name variables and constants in an understandable way "F" doesn't mean anything

Reply:
This is a misunderstanding, I'm importing the constant F_OK from module constants. It's a pre-defined file access constant which is used in file.access() method as access privilages constant.